src/angular/1-why-angular/why-angular.md

```diff
@@ -13,22 +13,22 @@ Some of the best things about Angular are that it is opinionated, has built-in t
 
 ### 1. An Opinionated Framework
 
-Developers have to make a lot of decisions on a daily basis, which can often create decision fatigue. The great part about using an opinionated framework is being able to shift your focus from build configs, high level architecture decisions,   to writing the actual client-specific functioning pieces of the application.
+Developers have to make a lot of decisions on a daily basis, which can often create decision fatigue. The great part about using an opinionated framework like Angular, is being able to shift your focus from build configs and high level architecture decisions, to writing the actual client-specific functioning pieces of the application. 
```

Comment:
unnecessary space at the end of the line (or is it a line break?)

Reply:
I think it's a space :P, I'll be sure to clean up trailing spaces~ plz block this PR for now

Reply:
Should be good now